### ISSUE TITLE
Keyboard navigation similar to Reddit Enchanment Suite

### ISSUE THREAD

**Issue by ddorstijn** [rystaf/mlmym#72]
The frontend looks awesome already. I have used Reddit old for many years and the experience feels similar. One thing I am missing (and I have yet to find a frontend that implements this) is keyboard navigation. I don't know how people do it, move the mouse so much all the time. I would really like to quickly go through all the items opening and closing images as I please.

I would help with implementing it but I don't know if I have the time to do so, or that it is something that is wished for in the project.

**Comment by jj777** [comment]
Yes! I would love A (Upvote) / D (Downvote) and X (Expand Image Inline)!

**Comment by PresidentPyrus** [comment]
I know this is an old issue, but it would also be really nice to be able to save (post) comments with ctrl+enter, like on Reddit and with [Lemmy-UI](https://github.com/LemmyNet/lemmy-ui).
